test/webpack.build.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs-extra');
```

Comment:
Personally, I'd just disable eslint for any test utility files, they have little to no bearing on code quality where it really matters.